tests/Doctrine/Tests/DBAL/Platforms/OraclePlatformTest.php

```diff
@@ -844,7 +841,7 @@ public static function getReturnsGetListTableColumnsSQL() : iterable
 WHERE    c.table_name = 'test'
 ORDER BY c.column_id
 SQL
-                ,
+,
```

Comment:
Is this expected?

Reply:
:thinking: no… it looks weird to me, but phpcs complains if I don't do it…

```
 844 | ERROR | [x] Expected 0 spaces before comma; 15 found
     |       |     (Squiz.Arrays.ArrayDeclaration.SpaceBeforeComma)
```

Looks like a bug…

Reply:
Hey look who's there when you type "heredoc nowdoc SpaceBeforeComma" on the phpcs repo! https://github.com/squizlabs/PHP_CodeSniffer/issues/2165

Reply:
I think a nice solution is to avoid putting the comma on its own line

Reply:
Ah in fact it's not nice at all, because it does not work on 7.3, reverting this.

Reply:
Reported here: https://github.com/squizlabs/PHP_CodeSniffer/issues/2951